frontend/src/pages/Components/Lesson/GetLessonById.jsx

```diff
@@ -28,11 +28,14 @@ const GetLessonById = (props) => {
             const response = await fetch(
                 `http://localhost:8080/lessons/${id}`
             );
+            //todo - to samo co w getLessons
```

Comment:
is it comment from code review by Konrad? is it still important for future development?